apps/communications/contacts/elements/details.html

```diff
@@ -1,7 +1,7 @@
 <element name="view-contact-details" extends="section">
   <template>
     <gaia-header id='details-view-header' action="back">
-      <h1 id='contact-name-title'></h1>
+      <h1 id='contact-name-title'><bdi class="ellipsis-dir-fix"></bdi></h1>
```

Comment:
nevermind.. it will probably make things harder and might cause undesired side effects. `<bdi>` is probably really the way to go.